src/webapp/components/survey-list/hook/useFilteredSurveys.ts

```diff
@@ -21,27 +21,32 @@ export function useSurveyList(
         resetCurrentCountryQuestionnaire,
         resetCurrentHospitalForm,
         resetCurrentWardRegister,
+        resetCurrentFacilityLevelForm,
     } = useCurrentSurveys();
 
     useEffect(() => {
+        //filters apply only to PPSSurveyForm
+        if (surveyFormType !== "PPSSurveyForm") {
```

Comment:
if this useEffect() continues to grow with more if-else it may be better in the future to take out functions

Reply:
Noted.